.goreleasermac.yml

```diff
@@ -0,0 +1,29 @@
+before:
```

Comment:
is it necessary to have this duplicate config? why not just ".goreleaser.yml"

Reply:
i'll remove .gorelasemac.yml, move it to current yml.

Reply:
linux arm64 is possible, i'll add it.